Formula/a/argc.rb

```diff
@@ -0,0 +1,21 @@
+class Argc < Formula
+  desc "Easily create and use cli based on bash script"
+  homepage "https://github.com/sigoden/argc"
+  url "https://github.com/sigoden/argc/archive/refs/tags/v1.14.0.tar.gz"
+  sha256 "4e1b3b01ef3dd590b1201a84d63b275ab65ce145793a1d15b57cfaa2149d3791"
+  license any_of: ["Apache-2.0", "MIT"]
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", *std_cargo_args
```

Comment:
can you also try to add the completion support? 

- https://github.com/sigoden/argc/pull/170

Reply:
The bash completion of argc requires bash version 5 or later, will this be an issue?

Reply:
I think it would be an issue, but we will see.

Reply:
Typically, argc is registered for completions along with the scripts written based on argc, such as `argc --argc-completions zsh mycmd1 mycmd2 ...`. Or use with https://github.com/sigoden/argc-completions

So initially I didn't add the completion.
I'm not sure if it's necessary to generate completion only for argc.

Reply:
yeah, if upstream supports the completion generation, then why not?